Document/0x05c-Reverse-Engineering-and-Tampering.md

```diff
@@ -1247,6 +1247,112 @@ The following approach can be used in order to patch the JavaScript file:
 
 #### Dynamic Instrumentation
 
+##### Information Gathering
+
+In this section we will learn about how to use Frida to obtain information about a running application.
+
+###### Getting Loaded Classes and their Methods
+
+In the Frida REPL to access Java runtime, `Java` command can be used to access information within the running app. Unlike `ObjC` command for iOS, for Java, the code need to be called via `Java.perform()` function. Thus, the best way to get list of loaded Java classes and their corresponding methods and field is to use Frida scripts. One such script is listed below.
+
+```
```

Comment:
Could you please give the Frida command to load the script and execute it for an app?

Reply:
I have added the command to be used just before the output.